.circleci/config.yml

```diff
@@ -62,14 +135,51 @@ jobs:
             git clone https://github.com/varnishcache/pkg-varnish-cache.git .
             git checkout << pipeline.parameters.pkg-commit >>
             tar cvzf debian.tar.gz debian --dereference
-            tar cvzf redhat.tar.gz redhat --dereference 
+            tar cvzf redhat.tar.gz redhat --dereference
             tar cvzf alpine.tar.gz alpine --dereference
+            mkdir -p ~/workspace
```

Comment:
same, the use of an extra directory seems a bit gratuitous?

Reply:
Maybe I don't quite understand how CircleCI's `persist_to_workspace` works.
Here is how I understood it:
```
            ...
            mkdir -p ~/workspace
            mv *.tar.gz ~/workspace
      - persist_to_workspace:
          root: ~/workspace
          paths:
            - debian.tar.gz
            - redhat.tar.gz
            - alpine.tar.gz
```

I create `~/workspace` as a folder that will contain the outcomes from all steps and all following steps may use any of the outcomes. This way is simpler to debug the steps - I just list the contents of `~/workspace` at the beginning of each step and it is easier to understand why some step fails.
Then I move the .tar.gz archives into it and tell CircleCI to export this folder. Maybe I can remove the `paths` parameter. I guess this way CircleCI will export the complete contents of the folder, not listing the files explicitly.

What do you suggest to simplify it ?

Reply:
ok, I understand the approach now. My problem is that `persist_to_workspace` and `store_artifacts` and used pretty much the same way, which is a bit confusing.

I guess that renaming `workspace` to `saved` or something like that would have reduced confusion, but we are down to bikeshedding now, so it's fine